packages/node_modules/@webex/plugin-meetings/src/metrics/config.js

```diff
@@ -135,6 +135,7 @@ export const eventType = {
   MUTED: 'client.muted',
   UNMUTED: 'client.unmuted',
   LEAVE: 'client.call.leave',
+  END: 'client.call.end',
```

Comment:
Previously was end event not being sent on leave ?

Reply:
this is new event i have created.